pkgs/tools/misc/ttfautohint/default.nix

```diff
@@ -1,23 +1,27 @@
-{ stdenv, lib, fetchurl, pkgconfig, freetype, harfbuzz, libiconv, qtbase, enableGUI ? true }:
+{
+  stdenv, lib, fetchurl, pkgconfig, autoreconfHook
+, freetype, harfbuzz, libiconv, qtbase
+, enableGUI ? true
+}:
 
 stdenv.mkDerivation rec {
-  version = "1.7";
+  version = "1.8.1";
   name = "ttfautohint-${version}";
 
   src = fetchurl {
     url = "mirror://savannah/freetype/${name}.tar.gz";
-    sha256 = "1wh783pyg79ks5qbni61x7qngdhyfc33swrkcl5r1czdwhhlif9x";
+    sha256 = "1yflnydzdfkr8bi29yf42hb6h6525a4rdid3w8qjfk8rpqh53pqj";
   };
 
   postPatch = ''
     substituteInPlace configure --replace "macx-g++" "macx-clang"
   '';
 
-  nativeBuildInputs = [ pkgconfig ];
+  nativeBuildInputs = [ pkgconfig autoreconfHook ];
 
   buildInputs = [ freetype harfbuzz libiconv ] ++ lib.optional enableGUI qtbase;
 
-  configureFlags = lib.optional (!enableGUI) "--with-qt=no";
+  configureFlags = [ ''--with-qt=${if enableGUI then "${qtbase}/lib" else "no"}'' ];
```

Comment:
Is the change to `configureFlags` necessary?

Reply:
No I don't think so. Any problems with that change?

Reply:
No, but it looks suspicious.

I have little time to review at the moment, but I noticed this PR due to the darwin build failure, whereas I knew that ttfautohint GUI used to work on darwin. I understand that master is in the bad shape now, but in general I expect that PRs that fail to build on the declared platforms are not merged. I had to fix `mysql.connector-c` in b41e5ece146a6357364ca2dd1a95ece6c22d257a and `ttfautohint` in d1204e65c4f4d0edfe6d395805e33abee3837439.

Reply:
Ohh sorry! I did not wanted to break stuff on Mac Os. I thought that the package failed before also :( 
The problem is that I have no Mac Os and can not do any debuggin/fixing :/

Reply:
@orivej: sorry!